Blog/templates/Blog/content.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+{% extends 'Blog/base.html' %}
```

Comment:
This is wrong inheritance. A child template should have only extends and blocks defined in parent.
```
{% extends 'Blog/base.html' %}
{% block xx %} {% endblock %}
{% block xx %} {% endblock %}
{% block xx %} {% endblock %}
```